### PR TITLE
removed leading footnote space, and more

### PR DESCRIPTION
**Pull request by pgmmpk**
1. Removed space before `<footnote>`
2. If `<footnote>` immediately follows a closing tag (like `</red>),  move it inside the tag

### CODE REVIEW

File: Apostol1989/chapters/05deyan.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
...
 <p><red>Коне́цъ па́сцѣ.</red></p>
 <p>ѳ҃ И҆ сїѧ̑ ре́къ, зрѧ́щымъ и҆̀мъ, взѧ́тсѧ: и҆ ѻ҆́блакъ под̾ѧ́тъ є҆го̀ ѿ ѻ҆́чїю и҆́хъ. і҃ И҆ ꙗ҆́кѡ взира́юще бѧ́хꙋ на нб҃о, и҆дꙋ́щꙋ є҆мꙋ̀, и҆ сѐ мꙋ́жа два̀ ста́ста пред̾ ни́ми во ѻ҆де́жди бѣлѣ̀, а҃і Ꙗ҆̀же и҆ реко́ста: мꙋ́жїе галїле́йстїи, что̀ стоитѐ зрѧ́ще на нб҃о; се́й і҆и҃съ вознесы́йсѧ ѿ ва́съ на нб҃о, та́кожде прїи́детъ, и҆́мже ѻ҆́бразомъ ви́дѣсте є҆го̀ и҆дꙋ́ща на нб҃о.</p>
-<p><red>(заⷱ҇ в҃)</red> в҃і <red>Т</red>огда̀ возврати́шасѧ <red>꙳</red><footnote lang="cu"><red>Въ понедѣ́льникъ свѣ́тлыѧ недѣ́ли:</red> Во днѝ ѻ҆́ны, возврати́шасѧ а҆пⷭ҇ли<red>꙳</red></footnote> во і҆ерꙋсали́мъ ѿ горы̀ нарица́емыѧ є҆леѡ́нъ, ꙗ҆́же є҆́сть бли́зъ і҆ерꙋсали́ма, сꙋббѡ́ты и҆мꙋ́щїѧ пꙋ́ть.</p>
+<p><red>(заⷱ҇ в҃)<footnote lang="cu"><red>Въ понедѣ́льникъ свѣ́тлыѧ недѣ́ли:</red> Во днѝ ѻ҆́ны, возврати́шасѧ а҆пⷭ҇ли<red>꙳</red></footnote></red> в҃і <red>Т</red>огда̀ возврати́шасѧ <red>꙳</red> во і҆ерꙋсали́мъ ѿ горы̀ нарица́емыѧ є҆леѡ́нъ, ꙗ҆́же є҆́сть бли́зъ і҆ерꙋсали́ма, сꙋббѡ́ты и҆мꙋ́щїѧ пꙋ́ть.</p>
```
Comment (typiconman) [inline Apostol1989/chapters/05deyan.xml]: Is this right -- to have the `<footnote>` inside of the `<red>`?

File: Apostol1989/chapters/05deyan.xml [hunk 2]
```diff
@@ -21,47 +21,47 @@
...
+<p><red>(заⷱ҇ д҃) [а҃]<footnote lang="cu"><red>Во вто́рникъ свѣ́тлыѧ недѣ́ли:</red> Во днѝ ѻ҆́ны, ста́въ пе́тръ<red>꙳</red></footnote></red> д҃і <red>С</red>та́въ же пе́тръ <red>꙳</red> со є҆динона́десѧтьми воздви́же гла́съ сво́й, и҆ речѐ и҆̀мъ: мꙋ́жїе і҆ꙋде́йстїи, и҆ живꙋ́щїи во і҆ерꙋсали́мѣ всѝ, сїѐ ва́мъ разꙋ́мно да бꙋ́детъ и҆ внꙋши́те глаго́лы моѧ̀: є҃і Не бо̀, ꙗ҆́коже вы̀ непщꙋ́ете, сі́и пїѧ́ни сꙋ́ть: є҆́сть бо ча́съ тре́тїй днѐ. ѕ҃і Но сїѐ є҆́сть рече́нное проро́комъ і҆ѡ́илемъ: з҃і И҆ бꙋ́детъ въ послѣ́днїѧ днѝ [глаго́летъ гдⷭ҇ь,] и҆злїю̀ ѿ дх҃а моегѡ̀ на всѧ́кꙋ пло́ть, и҆ прорекꙋ́тъ сы́нове ва́ши, и҆ дщє́ри ва́шѧ, и҆ ю҆́ноши ва́ши видѣ̑нїѧ ᲂу҆́зрѧтъ, и҆ ста́рцы ва́ши сѡ́нїѧ ви́дѧтъ. и҃і И҆́бо на рабы̀ моѧ̀, и҆ на рабы̑ни моѧ̀, во днѝ ѻ҆́ны и҆злїю̀ ѿ дх҃а моегѡ̀, и҆ прорекꙋ́тъ. ѳ҃і И҆ да́мъ чꙋдеса̀ на нб҃сѝ горѣ̀, и҆ зна́мєнїѧ на землѝ ни́зꙋ, кро́вь, и҆ ѻ҆́гнь, и҆ кꙋре́нїе ды́ма. к҃ Со́лнце преложи́тсѧ во тмꙋ̀, и҆ лꙋна̀ въ кро́вь, пре́жде да́же не прїитѝ дню̀ гдⷭ҇ню вели́комꙋ и҆ просвѣще́нномꙋ. к҃а И҆ бꙋ́детъ, всѧ́къ, и҆́же а҆́ще призове́тъ и҆́мѧ гдⷭ҇не, спасе́тсѧ.</p>
```
Comment (typiconman) [inline Apostol1989/chapters/05deyan.xml]: Above the `<footnote>` is placed after the "zachalo", but here it is placed after the verse. Is this right?

The point of these footnotes is to give the beginning words of the Zachalo, up until the asterisk, so perhaps the footnotes should apply to everything from `(заⷱ҇)` up until the asterisk?

Comment (pgmmpk) [reply Apostol1989/chapters/05deyan.xml]: Does not look right. My intention was just to move footnote close to the text, i.e. inside the `<red>` span. Looks like here my RegEx was too greedy and moved footnote too far forward.

**Comment by pgmmpk** [comment]
Cancelling PR as erroneous

**Pull request closed without merging**